### PR TITLE
Issue-330 Upgrade dependencies to support android 12

### PR DESCRIPTION
**Pull request by squallsama**
## Description

Upgrading play-services-ads to 20.2.0 with recent version of androidx.work:work-runtime:2.7.0-alpha05 that supports android 12

## Related Issues

https://github.com/googleads/googleads-mobile-flutter/issues/330

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [*] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [*] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [*] I read and followed the [Flutter Style Guide].
- [*] I signed the [CLA].
- [*] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [*] No, this is *not* a breaking change.

[issue database]: https://github.com/googleads/googleads-mobile-flutter/issues
[Contributor Guide]: https://github.com/googleads/googleads-mobile-flutter/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by squallsama** [comment]
@googlebot I signed it!

File: packages/google_mobile_ads/android/build.gradle [hunk 1]
```diff
@@ -31,8 +31,13 @@ android {
         disable 'InvalidPackage'
     }
     dependencies {
-      api 'com.google.android.gms:play-services-ads:20.1.0'
-      testImplementation 'junit:junit:4.12'
+      api 'com.google.android.gms:play-services-ads:20.2.0'
+      constraints {
+        implementation('androidx.work:work-runtime:2.7.0-rc01') {
```
Comment (westy92) [inline packages/google_mobile_ads/android/build.gradle:36]: FYI 2.7.0 is now stable.

Comment (squallsama) [reply packages/google_mobile_ads/android/build.gradle:36]: I'm going to update it to 2.7.0 and also sync with master again. Thanks!

Comment (squallsama) [reply packages/google_mobile_ads/android/build.gradle]: Done

**Comment by jjliu15** [comment]
Closing in favor of https://github.com/googleads/googleads-mobile-flutter/pull/423

**Comment by squallsama** [comment]
What the difference between my PR and this one ?

**Pull request closed without merging**